cmd/metricswatcher/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"os"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+
+	"github.com/mattermost/mattermost-load-test-ng/cmd/metricswatcher/config"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:           "metricswatcher",
+		RunE:          runMetricsWatcher,
+		PreRun:        config.SetupMetricsCheck,
+		SilenceErrors: true, // Since we're printing our logs, we don't need cobra to print the errors
+		SilenceUsage:  true, // For some reason cobra prints the usage when some error happens in Execute()
+	}
+
+	persistentFlags := rootCmd.PersistentFlags()
+	persistentFlags.StringP("config", "c", "", "path to the configuration file to use")
+	persistentFlags.StringP("queries", "q", "", "path to the JSON file with Prometheus queries")
+	cobra.MarkFlagRequired(persistentFlags, "queries")
+
+	if err := rootCmd.Execute(); err != nil {
+		os.Exit(1)
+	}
```

Comment:
I suggest we log the error here. Running without the required config file will fail without notifying the user.

Reply:
I moved the logger call there. I also re-enabled the viper settings I had disabled for error output. Some important errors were being silenced, so I considered it's better to have duplicated error output when errors happened than to have no output at all.